### PR TITLE
Add 2x2 cemetery

### PR DESCRIPTION
**Pull request by pisskop**
Creates 48 by 48 cemetery in the woods, themed on a quaint, well-to-do religious cemetery.  No specific religion, of course.

Adds an item list, and most of what I plan to use to create other cemeteries.

### CODE REVIEW

File: data/json/mapgen/cemetery_4square.json [hunk 1]
```diff
@@ -0,0 +1,375 @@
+[
+    {
+    "type" : "item_group",
+    "id" : "grave_memories",
+    "items":[
+      ["teddy", 15],
```
Comment (mugling) [inline data/json/mapgen/cemetery_4square.json]: alphabetical ordering is generally helpful in reducing future merge conflicts

**Comment by BorkBorkGoesTheCode** [comment]
May I have a copy of your cemetery?

**Comment by pisskop** [comment]
umm?  I thought you could copy it.

Yea.  Ill pm it.

**Comment by BorkBorkGoesTheCode** [comment]
Thank you!

**Comment by illi-kun** [comment]
I've tested this PR and here are some comments:

- Passable statues and gravestones are weird. For me, (at least) statues should be impassable.

- Mix of sidewalks and pavements looks flawed / inaccurate:
cemetery

**Comment by pisskop** [comment]
it looks like ive left rotate_true in there somewhere.  thats why the pavement looks wacky.


and statues is already integrated.  i could change that.

why would grave stones be impassable?  these are, at their tallest, the height of the average persons waist.  jumpable in all circumstances.

**Comment by illi-kun** [comment]
> statues is already integrated.

I didn't notice that. For me, it's reasonable to make it impassable but this is discussable.

> why would grave stones be impassable?

Yeah, you're right - it's actually comparable with open windows - it's possible to pass thru it with increased movecost, so no changes required.

**Comment by pisskop** [comment]
Im in favor of not letting people walk through statues.

With its dargray coloring and capital 'S' symbol it looks like they are meant to be significant entities, not busts or little plaques or something to that nature.

This:
<a href="http://imgur.com/rquVYzM"></a>

Is the worst kind of 'road and mapgen tile not syncing up' id expect.  I suppose I could either redesign the tile to make 2 entrances or include a 3-way road tile at the base, what would be [0, 2]?

**Comment by pisskop** [comment]
I could upload this extra tile if you would feel better about it, but any other pavement/grass inconsistencies are on the end of the games hardcoded roads, not me.

<a href="http://imgur.com/5qeA6mW"></a>

**Comment by Coolthulhu** [comment]
Those smooth rock walls look wrong, because they do not connect to stained glass windows or each other.

Stained glass windows need "CONNECT_TO_WALL" flag, the wall needs "AUTO_WALL_SYMBOL" flag.

**Comment by pisskop** [comment]
aye

**Pull request merged**